packages/wrangler/src/dev.tsx

```diff
@@ -555,6 +555,12 @@ function useWorker(props: {
     port,
   } = props;
   const [token, setToken] = useState<CfPreviewToken>();
+
+  // This is the most reliable way to reliably detect whether
```

Comment:
A bit tautological 😄 (and a split infinitive!! Bold, Kirk!)

```suggestion
  // This is the most reliable way to detect whether
```